internal/cmd/schema-registry/command_cluster_test.go

```diff
@@ -93,14 +98,34 @@ func (suite *ClusterTestSuite) SetupTest() {
 			return resp, nil, nil
 		},
 	}
+	suite.srcmClusterMock = &srcmv2mock.ClustersSrcmV2Api{
```

Comment:
It would be amazing if these unit tests could be converted to integration tests in this PR!

Reply:
they're already covered in inte-tests! Do you wish to remove them unit tests?

Reply:
As long as you're certain we're not losing any test coverage, yes! This would make maintenance easier in the long-term.